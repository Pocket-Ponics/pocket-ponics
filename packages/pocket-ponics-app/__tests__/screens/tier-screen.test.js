import React from 'react'
import { shallow } from 'enzyme'

import TierScreen from '../../screens/tier-screen'

describe('Tier Screen', () => {
	test('renders correctly', () => {
		global.greenhouses = {}
		global.plants = []
		const tree = shallow(
			<TierScreen navigation={{ getParam: jest.fn() }}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})