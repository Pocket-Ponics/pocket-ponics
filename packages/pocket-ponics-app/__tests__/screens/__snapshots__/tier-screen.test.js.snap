// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tier Screen renders correctly 1`] = `"<View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><View style=\\"flex:1\\"><Image style=\\"width:100%;height:175px;margin-top:30px;margin-bottom:30px;resize-mode:contain\\"></Image><Text style=\\"font-size:36px;font-weight:bold;margin-bottom:20px;color:#FFFFFF;text-align:center\\"></Text><View style=\\"flex-direction:row\\"><View style=\\"align-items:flex-start\\"><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">pH:</Text> </Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">Electrical Conductivity:</Text> </Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">Estimated Harvest:</Text> NaN/NaN/NaN</Text></View><View style=\\"padding-left:20px\\"><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\">Adjusting</Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\">Adjusting</Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"></Text></View></View></View></View>"`;
